skimage/feature/tests/test_canny.py

```diff
@@ -104,3 +104,13 @@ def test_invalid_use_quantiles(self):
 
         self.assertRaises(ValueError, F.canny, image, use_quantiles=True,
                           low_threshold=0.5, high_threshold=-100)
+
+    def test_dtype(self):
+        """Check that the same output is produced regardless of image dtype."""
+        image_uint8 = data.camera()[::50, ::50]
```

Comment:
Does it make sense to test with such a small image?

Reply:
No, you're right, It might be better to test with a full image instead. I used this as it was used in another test.

Reply:
I updated the test to use the full image instead.

Reply:
nice. Yeah, I guess for these correctness test it is important to have a meaningful result.